StandoutNews.php

```diff
@@ -38,7 +37,7 @@ private function standout_fetch_data()
         } catch (Exception $ex) {
             $news = null;
         }
-        return $news;
+        return $news->articles;
```

Comment:
If there is a exception above here news will be set to null, but we still return $news->articles.
That will not be defined when there is a exception.